packages/admin/cms-admin/src/generator/future/utils/stringManipulation.ts

```diff
@@ -0,0 +1,3 @@
+export const firstCharUpperCase = (str: string) => {
```

Comment:
Please use the change-case package here, which we already use for [camelCaseToHumanReadable](https://github.com/vivid-planet/comet/blob/main/packages/admin/cms-admin/src/generator/future/utils/camelCaseToHumanReadable.ts)

Reply:
Cool 👍🏼 
Done in fc799fba26bec4579cabf3f723a34f65cb85075b